### DISCUSSION TITLE
How farm know each remote harvester's plots separately?

### DISCUSSION THREAD

**Discussion by 21497724** [Chia-Network/chia-blockchain#6156] [General]
I have set up a farming on many machines following the https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines.
Is there any way to now each remote harvester's plots in the full node site?

**Comment by 21497724** [comment]
Anyone knows?

**Comment by XIII39** [comment]
Set the logging to INFO the the config.yaml on your farmer.

Search the log file for your local IP addresses, in Ubuntu I would do `cat ~/.chia/mainnet/log/debug.log | grep 192.168.0` since this set of IP addresses are my harvesters.

You should see something like;

```
-> new_signage_point_harvester to peer 192.168.0.XXX and
<- farming_info from peer 192.168.0.XXX
```

Here you can see the farmer sending the data to the harvester and the harvester responding to the farmer.

**Reply by XIII39** [reply]
Also don't forget the edit the config.yaml file on each harvester.

Set the `harvester: farmer_peer` to your farmer IP address and add the local `plot_directories `

**Reply by KipourosV** [reply]
Yes but what about aggregated stats for all harvesters on GUI.
Is that something that will be implemented?

**Reply by racabu** [reply]
now you have to use log level as DEBUG in order for this message to appear

**Reply by 21497724** [reply]
Yes，set to debug level, did see the new_signage_point_harvester information, however can't get how many plots in each harvester, this is what I want to konw.

**Comment by 21497724** [comment]
Anyone knows? how full node know each connected harvester's plots number?

**Reply by firmicus** [reply]
I'm also wondering...

**Reply by 21497724** [reply]
the 1.1.7 version still not
